content/blog/2024/07/kubewarden-1-15-release.md

```diff
@@ -0,0 +1,104 @@
+consistency and reduces the chances of configuration drift. By defining these
+rules once and applying them universally, users can achieve a more reliable and
+predictable policy enforcement setup.
+
+:::warning
+The affinity configuration previously present in the default policy server values 
+has been removed. Then, if you were using it, you need to migrate your configuration
+:::
+
+## Technical Debt Reduction in Kubewarden Controller 
+
+In v1.15, significant efforts have been made to reduce technical debt in the
+Kubewarden controller, ensuring better performance, easier maintenance, and a
+more robust foundation. Key changes include:
+
+- **Updated Golang Linter**: The stricter linter catches potential issues
+  early, ensuring higher code quality.
+- **Removed --default-policy-server CLI Flag**: The default policy server name
+  is now always "default," simplifying configuration.
+- **Updated to Kubebuilder 4.0.0**: The controller code now uses Kubebuilder
+  4.0.0.
+- **Refactored Reconciler Code**: The code is split into multiple
+  subreconcilers following the Subreconciler pattern, enhancing manageability
+  and modularity.
+- **File/Package Renaming**: Files and packages have been renamed for better
+  readability and navigation.
+
+These changes improve functionality and set the stage for more streamlined
+development and maintenance, reflecting a commitment to maintaining a
+high-quality codebase.
+
+## Policy Grouping Code Implementation 
+
+The v1.15 release marks the beginning of implementing [RFC 0020: Policy
+Group](https://github.com/kubewarden/rfc/blob/main/rfc/0020-policy-group.md),
+starting with changes to the policy server. This feature allows users to create
+complex policies by combining simpler ones, leveraging existing policies to
+avoid custom builds from scratch. The motivation is to increase reusability,
+reduce cognitive load, and enable the creation of custom policies through a
+DSL-like configuration, thus improving policy management efficiency and
+flexibility.
+
+## CEL Policy Enhancements 
+
+The Common Expression Language (CEL) policies in Kubewarden have received two
+notable enhancements:
+
+- **OCI Capability for Fetching Configs**: Now includes an OCI capability to
+  fetch container image configurations, making it easier to retrieve and apply
+  policy configurations from OCI-compliant registries.
+- **Lazy Variable Evaluation**: Introduces lazy variable evaluation, deferring
+  variable evaluation until needed, thus improving performance and efficiency
+  by reducing unnecessary computations.
+
+## Extended policies CRDs with MatchConditions 
+
+Lastly, the ClusterAdmissionPolicy and AdmissionPolicy CRDs have been extended
+to include `matchConditions`. This enhancement provides users with more
+granular control over policy enforcement by allowing them to specify conditions
+under which policies should be applied. Therefore, users can ensure that the
+right rules are enforced in the right scenarios.
```

Comment:
```suggestion
right rules are enforced in the right scenarios.

The propagation of `matchConditions` can be done only on these
versions of Kubernetes:

- 1.26 or earlier: not available. The `matchConditions` are ignored
- 1.27, 1.28, 1.29: available only when the `AdmissionWebhookMatchConditions`
  [feature gate](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/) is enabled. Otherwise, the `matchConditions` are ignored
- 1.30 or later: always available, no action required
```